_data/conference/202503.yaml

```diff
@@ -23,10 +23,12 @@ agenda:
     speaker:
       - name: jmesnil
     links:
-      - title: Chat
-        url: https://wildfly.zulipchat.com/#narrow/channel/492672-Mar-2025-Mini-Conference/topic/Thread.3A.20Quick.20prototyping.20with.20WildFly/with/507327271
-      - title: Feedback
-        url: https://forms.gle/GgbBoPC4AYtnLiz87
+      - title: "Chat"
+        url: "https://wildfly.zulipchat.com/#narrow/channel/492672-Mar-2025-Mini-Conference/topic/Thread.3A.20Quick.20prototyping.20with.20WildFly/with/507327271"
+      - title: "Slides"
+        url: "https://www.wildfly.org/assets/conference/202503-ai-mcp-server.pdf"
```

Comment:
not the right one, it should be https://www.wildfly.org/assets/conference/202503-jbang-prototyping.pdf